pym/bob/utils.py

```diff
@@ -179,6 +179,9 @@ def cmp(l, r):
     return ret
 
 
+def isMsys():
```

Comment:
I think this is wrong because Cygwin is not exactly MSYS2. It should only return `True` if we're really running in MSYS2.

Reply:
i have no idea about cygwin, that's a msys, with a windows cross gcc, isn't it?